admin/config.yml

```diff
@@ -38,6 +38,19 @@ collections:
         fields:
           - { name: body, label: Description, widget: markdown }
 
+      - label: Call for Macroscopes
+        name: callForMacroscopes
+        file: content/site/callForMacroscopes.md
+        preview_path: call-for-macroscopes
+        fields:
+          - { name: lastIteration, label: Previous Macroscope Iteration, widget: number, min: 11, max: 16, default: 16 }
```

Comment:
I see what you did there, but max should be 20 since we plan on having 10 iterations of macroscopes (unless I missed something)

Reply:
The page is straightforward. The images across the top represent the previous iteration of microscopes. 
The CMS should make it possible for the user to select the previous iteration to change the banner at will. 
Currently, those options are IT 11 through IT 16. The iteration that should be shown on the site is IT 16.

Above is the description I got from Leonard. 
The last iteration in content is 16 so I set it to 16.

Reply:
That's fine, but you have the _max_ set to 16 which means future maintainers will not be able to set it higher

Reply:
That's fine, but you have the max set to 16 which means future maintainers will not be able to set it higher

Reply:
changed max value to 20